en.txt

```diff
@@ -0,0 +1,61 @@
+History:
+	I mostly have experience with linux (servers, desktops), unix, c programming
+	language
+	I like to discover new, innovative solutions for various IT problems
+	I like to use interesting open-source solutions, I am looking for interesting ones
+	ideas, and I try to implement them myself, or sometimes improve them.
+	I have worked briefly in several startups. I am active in cooperation.
+	What I did in startups:
+	- start-up 1: I wrote Python automation scripts for various tasks (mostly
+	related to web-scrap).
+	Some real examples:
+	- The bot is designed to find bicycles of a specific price, in particular locations.
+	When found, send messages to Telegram.
+	- The bot is designed to find the most popular Reddit posts, and according to the topic, make 10 min
+	video montage to upload them to different YouTube channels
+	- start-up 2: I worked with embedded Linux in the yacto project.
+	I was selling WordPress installation services.
```

Comment:
The sentence should point to the past, and how many WordPress services you have install.